src/components/Resume/ResumeListApi.js

```diff
@@ -0,0 +1,78 @@
+import axios from "axios";
+
+const instance = axios.create({
+  baseURL: "https://jsonplaceholder.typicode.com",
+});
```

Comment:
I think we do not need this anymore, since you are working with hard code `result`. Same for line 10